### PR TITLE
Improve some docs

### PR DESCRIPTION
**Pull request by italolelis**
## What does this PR do?

- Improves documentation

### CODE REVIEW

**Comment by zeelax** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/161/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/161/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/161/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

File: docs/quick_start/add_auth.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Add Authentication
+
+When configuring your API you can choose between different authentication methods, these are:
+
+* Basic/Digest Authentication
+* OAuth 2.0
+* JWT
+
+We tried to desing Janus in a way that the authentication provider are simple to setup
```
Comment (rafaeljesus) [inline docs/quick_start/add_auth.md]: `tried to design`

File: docs/quick_start/add_auth.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        }
+    }
+}
+```
+
+So, what we've done here? 
+
+1. The first thing is to give a `name` for the oAuth2 server. 
+2. Within `oauth_endpoints` we setup only one endpoint for this example, which is the `token`. If you look closely you will see that the `oauth_endpoints.token` is just a proxy configuration, exactly the same that we used to configure our first endpoint.
+3. We've defined a `token_strategy`. Here you can choose between `jwt` or `storage`, storage means that Janus will be in charge of storing and managing (expiring, refreshing, etc) the tokens once they are returned from your oauth provider. JWT means that Janus will only check for expiration and secret of the tokens, but it wont store them.
```
Comment (rafaeljesus) [inline docs/quick_start/add_auth.md:51]: `won't`

**Review by rafaeljesus — CHANGES_REQUESTED** [review]
small typo

**Review by rafaeljesus — APPROVED** [review]

**Comment by rafaeljesus** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/161/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/161/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Pull request merged**